### PR TITLE
STABLE.md: 4.15.x is now stable

### PR DESCRIPTION
**Pull request by UTsweetyfish**
This pull request tries to merge into `4.15.x` branch.

### CODE REVIEW

File: STABLE.md [hunk 1]
```diff
@@ -2,10 +2,8 @@
 
 The following stable branches are kindly maintained by trusted volunteers:
 
--  4.14.x
+-  4.15.x
    -  git
-      -  [main](https://www.alejandro-colomar.es/src/alx/shadow/stable/shadow.git/log/?h=4.14.x)
```
Comment (alejandro-colomar) [inline STABLE.md:7]: The main repository for the stable branch is still that one, github being a mirror.  You can s/14/15/ there.

Comment (alejandro-colomar) [reply STABLE.md:7]: 4.15.1 is a glitch in the 4.15.x series.  :)

Comment (UTsweetyfish) [reply STABLE.md:7]: > The main repository for the stable branch is still that one, github being a mirror.

Ahh, just saw the certificate is self-signed. I mistakenly thought the main one is unmaintained!

Comment (alejandro-colomar) [reply STABLE.md:7]: It's documented here:
<https://www.alejandro-colomar.es/ssl>  :)

**Review by alejandro-colomar — APPROVED** [review]
```
Reviewed-by: Alejandro Colomar <alx@kernel.org>
```

Thanks!

**Comment by alejandro-colomar** [comment]
I'll let @hallyn or @ikerexxe merge this.

**Review by ikerexxe — APPROVED** [review]
LGTM! Thanks for the patch.

File: STABLE.md (hunk 1 above)
Comment (UTsweetyfish) [reply STABLE.md:7]: > It's documented here: https://www.alejandro-colomar.es/ssl :)

Such a good idea!

**Pull request merged**